dpsSettings.template.json

```diff
@@ -2,6 +2,5 @@
     "provisioningHost": "global.azure-devices-provisioning.net",
     "idScope": "",
     "primaryKey": "",
-    "dpsConnectionString": "",
-    "edgeDeviceConnectionString": ""
+    "dpsConnectionString": ""
```

Comment:
serviceConnectionString instead of dpsConnectionString

Reply:
Good point. Changed.